.github/workflows/selftest-linux.yml

```diff
@@ -80,3 +80,57 @@ jobs:
           pkgDir: testdata/dist # PKGDir must exist or docker will fail to `COPY` things
           stagingRepo: false
           distros: ${{ matrix.distro }}
+
+  linux-remote-specific-version-staging:
+    runs-on: ubuntu-latest
+    name: Remote linux self-test on specific staging Repo
+    strategy:
+      matrix:
+        distro: [ ubuntu, suse, rockylinux ]
+    steps:
+      - uses: actions/checkout@v2
+      - id: nri_f5_last
+        run: |
+          # We get the version from the latest release tag in nri-f5's repository 
+          version=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/newrelic/nri-f5/releases/latest \
+            | grep -oP "(?<=v)(0|[1-9]\d*)\.(0|[1-9]\d*).(0|[1-9]\d*)")
+
+          echo "VERSION=${version}" >> $GITHUB_OUTPUT
+      - name: Test staging package on ${{ matrix.distro }}
+        uses: ./linux
+        with:
+          tag: ${{ steps.nri_f5_last.outputs.VERSION }}
+          repoVersion: ${{ steps.nri_f5_last.outputs.VERSION }}
```

Comment:
The test would pass if we remove `repoVersion` field, wouldn't it? (As we are using the latest version which is the one is checked by default). Would it make sense to hard-code a previous version? (The test would be broken if that hard-coded version disappears, but we would had a use-case here the new field is needed).

Reply:
Yes, it would, I thought the same at first but went with this option and I knew (and wanted) I was going to get a comment in the review. We can either hard code it or do something like this:

`stagingRepo="http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent"
curl "$stagingRepo/linux/apt/dists/bionic/main/binary-amd64/Packages" \
     | grep -oP "(?<=nri-f5_)(0|[1-9]\d*)\.(0|[1-9]\d*).(0|[1-9]\d*)" | head -1`
     
The problem with the first approach is that version could disappear, and with my code approach, bash approach, it would fail if ubuntu's bionic is removed.

Reply:
As I see it, hard-coding can be good enough: it is really simple and we can also add a comment to clarify the reason why we hard-coded it (and how to fix it if the version disappears). The second approach wouldn't be affected if as specific version is removed, but as you pointed out, it will also fail when ubuntu's bionic is deprecated...

Reply:
Yes, hard codint is probably the best option

Reply:
I've modified it as you comment.